apollo/octopus/src/main/graphql/com/hedvig/android/apollo/octopus/graphql/flowTerminationDateNext.graphql

```diff
@@ -0,0 +1,21 @@
+mutation FlowTerminationDateNext($context: FlowContext!, $input: FlowTerminationDateInput!) {
+  flowTerminationDateNext(context: $context, input: $input) {
+    context
+    currentStep {
+      ... on FlowTerminationDateStep {
+        id
+        minDate
+        maxDate
+      }
```

Comment:
So these 3 we can try turning into a fragment to see if this gives us an easier experience working with it right?